bazel/repository_locations.bzl

```diff
@@ -601,6 +601,30 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2021-12-24",
         cpe = "N/A",
     ),
+    com_github_intel_isa_l = dict(
+        project_name = "isa-l",
+        project_desc = "Intel(R) Intelligent Storage Acceleration Library",
+        project_url = "https://github.com/intel/isa-l",
+        version = "2.30.0",
+        sha256 = "bcf592c04fdfa19e723d2adf53d3e0f4efd5b956bb618fed54a1108d76a6eb56",
+        strip_prefix = "isa-l-{version}",
+        urls = ["https://github.com/intel/isa-l/archive/v{version}.tar.gz"],
+        use_category = ["controlplane", "dataplane_core"],
+        release_date = "2020-11-07",
+        cpe = "N/A",
+    ),
+    com_github_netwide_assembler_nasm = dict(
+        project_name = "nasm",
+        project_desc = "NASM, the Netwide Assembler",
+        project_url = "https://github.com/netwide-assembler/nasm",
+        version = "nasm-2.15.05",
+        sha256 = "9182a118244b058651c576baa9d0366ee05983c4d4ae1d9ddd3236a9f2304997",
+        strip_prefix = "{version}",
+        urls = ["https://www.nasm.us/pub/nasm/releasebuilds/2.15.05/{version}.tar.gz"],
```

Comment:
Can we switch this to Github so dependency tracking will work?
`https://github.com/netwide-assembler/nasm/archive/refs/tags/nasm-2.15.05.tar.gz`

Reply:
For some time the Windows toolchain installs nasm as the preferred compiler on that platform, should we simply catch up the ubuntu toolchain and advise that nasm is a prerequisite?

Reply:
> Can we switch this to Github so dependency tracking will work? `https://github.com/netwide-assembler/nasm/archive/refs/tags/nasm-2.15.05.tar.gz`

The package of this link seems a little different, we may need add a patch to fix it.

Reply:
> For some time the Windows toolchain installs nasm as the preferred compiler on that platform, should we simply catch up the ubuntu toolchain and advise that nasm is a prerequisite?

I think it is good to slove dep issue, because many libs use `nasm` as dep, like [this](https://github.com/envoyproxy/envoy/pull/22717#issuecomment-1218216924).

Reply:
> For some time the Windows toolchain installs nasm as the preferred compiler on that platform, should we simply catch up the ubuntu toolchain and advise that nasm is a prerequisite?

@phlax Hi, could we add `nasm` to ubuntu toolchain? So there is no need to add external_dep in bazel.

Reply:
@giantcroc sorry i missed this previously

it would be far more preferable to add in the bazel toolchain - as that means it is dealt with as a proper dependency, gets dependency checked, updated, etc and is visible as being a (build) dep. It also means that the build is much more deterministic across platforms/archs, and makes building more portable to them.

Reply:
> 

OK, I am tring to contact with the developers of nasm to improve the security score of nasm github repo.